repair/row_level.cc

```diff
@@ -1034,6 +1042,7 @@ class repair_meta {
 
     future<uint64_t> get_estimated_partitions() {
       return with_gate(_gate, [this] {
+        // FIXME: for _mixed_shard_optimization
```

Comment:
I don't understand this comment.

Reply:
We need to adjust the estimation to count the partitions on all the shards for the given token range in case the mixed_shard_optimization is on. It is trivial. I simply do not have time to cover it yet. And the estimation is not very important with the recent filter rebuild work.

Reply:
Repair coordinator knows when it runs in mixed shard mode, so shouldn't we enable the optimization by default in that mode? I say this without really digging into your series, is there any reason to not do that?

Reply:
I mentioned here https://github.com/scylladb/scylladb/pull/19783#discussion_r1683590927